src/main/groovy/com/epam/petclinic/plugin/extensions/QualityAwareJavaExtension.groovy

```diff
@@ -0,0 +1,17 @@
+package com.epam.petclinic.plugin.extensions
+
+/**
+ * Comment here.
+ *
+ * Date: 5/17/2017
+ *
+ * @author Stanislau Halauniou
+ */
+class QualityAwareJavaExtension {
+    public static final String NAME = 'javaQuality'
+
+    String checkstyleToolVersion = '7.1'
+    String pmdToolVersion = '5.5.1'
+    String findbugToolVersion = '3.0.1'
+    String checkstyleSupressionPath
```

Comment:
I need to detect if this property is overridden or not,  different implementations.